HomebrewFormula/welle.io.rb

```diff
@@ -9,11 +9,12 @@ class WelleIo < Formula
   depends_on "qt"
   depends_on "fftw"
   depends_on "faad2"
-  depends_on "librtlsdr"
+  depends_on "soapysdr"
+  depends_on "soapyuhd"
```

Comment:
Are there other soapy device drivers you could optionally depend on? What comes to mind: hackrf, limesdr, plutosdr, maybe others